back/engines/commercial/user_custom_fields/app/controllers/user_custom_fields/web_api/v1/stats_users_controller.rb

```diff
@@ -9,7 +9,17 @@ class StatsUsersController < ::WebApi::V1::StatsController
 
         def users_by_age
           age_stats = AgeStats.calculate(find_users)
-          render json: age_stats, serializer: AgeStatsSerializer, adapter: :attributes
+
+          render json: raw_json({
+            total_user_count: age_stats.total_user_count,
```

Comment:
If you look at the serializer, it maps this to `user_count`, so replace `total_user_count ` by `user_count ` here.

Reply:
Ah, missed that- thanks!